pg_lakehouse/README.md

```diff
@@ -0,0 +1,213 @@
+
+```sql
+CREATE FOREIGN DATA WRAPPER local_file_wrapper HANDLER local_file_fdw_handler VALIDATOR local_file_fdw_validator;
+CREATE SERVER local_file_server FOREIGN DATA WRAPPER local_file_wrapper;
+
+-- Replace the dummy schema with the actual schema of your file data
+CREATE FOREIGN TABLE local_file_table (x INT)
+SERVER local_file_server
+OPTIONS (path 'file:///path/to/file.parquet', extension 'parquet');
+```
+
+#### Local File Table Options
+
+- `path` (required): Must start with `file:///` and point to the location of your file. The path should end in a `/` if it points to a directory of partitioned Parquet files.
+- `extension` (required): One of `avro`, `csv`, `json`, and `parquet`.
+- `format`: One of `delta` or `iceberg`. If omitted, no table format is assumed.
+
+### Amazon S3
+
+```sql
+CREATE FOREIGN DATA WRAPPER s3_wrapper HANDLER s3_fdw_handler VALIDATOR s3_fdw_validator;
+CREATE SERVER s3_server FOREIGN DATA WRAPPER s3_wrapper
+OPTIONS (
+    region 'us-east-1',
+    url 's3://path/to/bucket'
+    access_key_id 'XXXXXXX'
+    secret_access_key 'XXXXXXX'
+);
+
+-- Replace the dummy schema with the actual schema of your file data
+CREATE FOREIGN TABLE local_file_table (x INT)
+SERVER s3_server
+OPTIONS (path 's3://path/to/file.parquet', extension 'parquet');
+```
+
+#### S3 Server Options
+
+- `region` (required): AWS region, e.g. `us-east-1`
+- `url` (required): Path to the S3 bucket, starting with `s3://`
+- `access_key_id`: AWS access key ID
+- `secret_access_key`: AWS secret access key
+- `endpoint`: The endpoint for communicating with the S3 instance. Defaults to the [region endpoint](https://docs.aws.amazon.com/general/latest/gr/s3.html). For example, can be set to `http://localhost:4566` if testing against a Localstack instance.
+- `session_token`: Sets the AWS session token
+- `allow_http`: If set to `true`, allows both HTTP and HTTPS endpoints. Defaults to `false`.
+- `skip_signature`: If set to `true`, will not sign requests. This is useful for connecting to public S3 buckets. Defaults to `false`.
+
+#### S3 Table Options
+
+- `path` (required): Must start with `s3://` and point to the location of your file. The path should end in a `/` if it points to a directory of partitioned Parquet files.
+- `extension` (required): One of `avro`, `csv`, `json`, and `parquet`.
+- `format`: One of `delta` or `iceberg`. If omitted, no table format is assumed.
+
+## Data Movement
+
+### To Postgres
+
+Moving data from a data lake into Postgres can be done in a single query.
+
+```sql
+-- Assumes you have created the trips table from the quickstart
+CREATE TABLE trips_copy AS SELECT * FROM trips;
+```
+
+### To Data Lake
+
+Writes to data lakes are not yet supported but are coming soon.
+
+## Development
+
+### Install Rust
+
+To develop the extension, first install Rust v1.77.2 using `rustup`. We will soon make the extension compatible with newer versions of Rust:
```

Comment:
You can remove the "newer versions of Rust". v1.77.2 is the latest version, this was an artifact of being stuck on an old Rust version earlier due to Tantivy